src/easypost.js

```diff
@@ -51,63 +50,34 @@ export const METHODS = {
   DELETE: 'del',
 };
 
-export const RESOURCES = {
-  Address,
-  ApiKey,
-  Batch,
-  Billing,
-  Brand,
-  CarrierAccount,
-  CarrierType,
-  CustomsInfo,
-  CustomsItem,
-  EndShipper,
-  Event,
-  Insurance,
-  Order,
-  Parcel,
-  PaymentMethod,
-  Pickup,
-  Rate,
-  Refund,
-  Referral,
-  Report,
-  ScanForm,
-  Shipment,
-  Tracker,
-  User,
-  Webhook,
+export const SERVICES = {
+  Address: AddressService,
+  ApiKey: ApiKeyService,
+  Batch: BatchService,
+  BetaReferralCustomer: BetaReferralCustomerService,
+  Billing: BillingService,
+  CarrierAccount: CarrierAccountService,
+  CarrierType: CarrierTypeService,
+  CustomsInfo: CustomsInfoService,
+  CustomsItem: CustomsItemService,
+  EndShipper: EndShipperService,
+  Event: EventService,
+  Insurance: InsuranceService,
+  Order: OrderService,
+  Parcel: ParcelService,
+  Pickup: PickupService,
+  Rate: RateService,
+  ReferralCustomer: ReferralCustomerService,
+  Refund: RefundService,
+  Report: ReportService,
+  ScanForm: ScanFormService,
+  Shipment: ShipmentService,
+  Tracker: TrackerService,
+  User: UserService,
+  Webhook: WebhookService,
 };
 
-export const PROP_TYPES = {
-  addressPropTypes,
-  apiKeyPropTypes,
-  batchPropTypes,
-  billingPropTypes,
-  brandPropTypes,
-  carrierAccountPropTypes,
-  carrierTypePropTypes,
-  customsInfoPropTypes,
-  customsItemPropTypes,
-  endShipperPropTypes,
-  eventPropTypes,
-  insurancePropTypes,
-  orderPropTypes,
-  parcelPropTypes,
-  paymentMethodPropTypes,
-  pickupPropTypes,
-  ratePropTypes,
-  referralPropTypes,
-  refundPropTypes,
-  reportPropTypes,
-  scanFormPropTypes,
-  shipmentPropTypes,
-  trackerPropTypes,
-  userPropTypes,
-  webhookPropTypes,
-};
-
-export default class API {
+export default class EasyPostClient {
   constructor(key, options = {}) {
     const { useProxy, timeout, baseUrl, superagentMiddleware, requestMiddleware } = options;
```

Comment:
Question: Could we add `PropTypes` to validate these parameters?

Reply:
There is a follow up task for this